roles/openshift_logging_curator/files/config.yaml

```diff
@@ -0,0 +1,16 @@
+client:
+  hosts:
+  - ${ES_HOST}
```

Comment:
Why are the hots a list and the port a single value? Are we guaranteed to always use the same port on all hosts? Should hosts be a combination of  host and port?

Reply:
I think that's the nature of the distributed algs that es uses. It's defined by elastic anyway.